core/controlplane/config/templates/stack-template.json

```diff
@@ -461,6 +488,11 @@
             "Key": "Name",
             "PropagateAtLaunch": "true",
             "Value": "{{$.ClusterName}}-{{$.StackName}}-kube-aws-etcd-{{$etcdIndex}}"
+          },
+          {
+            "Key": "kube-aws:role",
```

Comment:
That's possible. I just prefer the dedicated key though because then it won't be affected by changes to `Name` and the etcdadm script becomes a bit simpler